dockerplugin.py

```diff
@@ -52,8 +52,33 @@ def _d(d):
     return ','.join(['='.join(p) for p in d.items()])
 
 
+def str_to_bool(value):
+    """Python 2.x does not have a casting mechanism for booleans.  The built in
+    bool() will return true for any string with a length greater than 0.  It
+    does not cast a string with the text "true" or "false" to the
+    corresponding bool value.  This method is a casting function.  It is
+    insensetive to case and leading/trailing spaces.  An Exception is raised
+    if a cast can not be made.
+    """
+    if str(value).strip().lower() == "true":
+        return True
+    elif str(value).strip().lower() == "false":
+        return False
+    else:
+        raise Exception("Unable to cast value (%s) to boolean" % value)
+
+
 def emit(container, dimensions, point_type, value, t=None,
          type_instance=None):
+    log.info(("Value parameters to be emitted: \n container : {c} "
```

Comment:
Use single quotes (that's the convention we've been using in this file). Applies to all your added messages/strings.